handlers/friday.go

```diff
@@ -0,0 +1,68 @@
+package handlers
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"time"
+
+	"io"
+	"net/http"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+var (
+	fridayTrigger = "sextou"
+	fridayGifUrl  = "https://media.tenor.com/zGlEbV_bTnIAAAAC/kowalski-familia.gif"
+)
+
+func FridayHandler(s *discordgo.Session, m *discordgo.MessageCreate) {
+	if !strings.Contains(m.Content, fridayTrigger) {
+		return
+	}
+
+	message := &discordgo.MessageSend{
+		Files: []*discordgo.File{},
+	}
+
+	switch time.Now().Weekday() {
+		case time.Friday:
+			message.Content = "Sextouu família"
+			message.Files = append(message.Files, processGifUrl(fridayGifUrl))
+		default:
+			message.Content = fmt.Sprintf("Calma família ainda não é sexta! Falta %d dia(s)", daysRemainingToFriday())
```

Comment:
You could select a gif like this and send with the message:

![Quinta-feira](https://media.tenor.com/xkeizGFocwYAAAAd/emojo-sexta.gif)

Or this:

![Falta pouco](https://media.tenor.com/qzpWHIWnMywAAAAC/a-little-bit.gif)